glib/src/lib.rs

```diff
@@ -1,61 +1,53 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
-//! # **glib**, **gobject** and **gio** bindings for Rust
+//! # Rust GLib and GObject bindings
 //!
-//! This library contains
-//!
-//! - bindings to some essential GLib, GObject, GIO types and APIs,
-//!
-//! - common building blocks used in both handmade and machine generated
-//! bindings to GTK+ and other GLib-based libraries.
+//! This library contains bindings to GLib and GObject types and APIs as well as
+//! common building blocks used in both handmade and machine generated
+//! bindings to GTK and other GLib-based libraries.
 //!
 //! It is the foundation for higher level libraries with uniform Rusty (safe and
 //! strongly typed) APIs. It avoids exposing GLib-specific data types where
 //! possible and is not meant to provide comprehensive GLib bindings, which
 //! would often amount to duplicating the Rust Standard Library or other utility
 //! crates.
 //!
-//! The library is a work in progress: expect missing functionality and breaking
-//! changes.
+//! The library is a work in progress. Expect breaking changes between releases.
```

Comment:
Is this still true?

Reply:
We have that in all the crates. I think it should be written a bit differently. There will be breaking changes but they should be rather minimal. Not very different to those crates that release a new major version every half year and are considered stable because they're >= 1.0.

Reply:
I guess let's handle that another time as we have the same comment on all the crates :)

Reply:
Yes, can you create an issue for that?

Reply:
I have posted gtk-rs/release#147 for this.

Reply:
Thanks Sophie :)

Reply:
https://github.com/gtk-rs/release/issues/147